test/functionalTest/cases/3121_missing_or_empty_metadata_values/entitiy_creation.test

```diff
@@ -0,0 +1,360 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Missing or empty metadata values are not allowed with POST /v2/entities/id/attrs
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/entities, to create Room1 with temperature (no metadata)
+# 02. GET /v2/entities/Room1
+# 03. POST /v2/entities/Room1/attrs, to add metadata with "" value to temperature and add pressure attribute with metadata with no value
+# 04. GET /v2/entities/Room1
+# 05. POST /v2/entities/Room1/attrs, to update temperature with metadata with no value, pressure with metadata with regular value and add humidity attribute with metadata with "" value
+# 06. GET /v2/entities/Room1
+# 07. POST /v2/entities, to create Room2 with temperature having two metadata one with "" value and another without value
+# 08. GET /v2/entities/Room2
+# 09. PUT /v2/entities/Room2/attrs/temperature/value , updating the value of Room2 temperature attribute.
```

Comment:
New description are good! :)

However, please include them also in the echo lines for each step. I mean change for instance:

```
 echo "03. POST /v2/entities/Room1/attrs." 
```
to 
```
echo '03. POST /v2/entities/Room1/attrs, to add metadata with "" value to temperature and add pressure attribute with metadata with no value'
```

Reply:
fixed in dc6cf02